homeassistant/components/humidifier/__init__.py

```diff
@@ -45,7 +46,13 @@
     DOMAIN,
     MODE_AUTO,
     MODE_AWAY,
+    MODE_BABY,
```

Comment:
It feels wrong including new modes in the same PR

Reply:
They are not new. It's just importing them to `__init__.py` so (in my case) I can import them from root for tests and integrations should also import them from there, and not from `.const`

Reply:
You are right, sorry